security/middleware.py

```diff
@@ -861,14 +853,14 @@ def process_existing_session(self, request):
         start_time = session[self.START_TIME_KEY]
         last_activity_time = session[self.LAST_ACTIVITY_KEY]
 
-        logger.debug("Session %s started: %s" % (
-            session.session_key,
-            start_time,
-        ))
-        logger.debug("Session %s last active: %s" % (
-            session.session_key,
-            last_activity_time,
-        ))
+        logger.debug("Session %s started: %s",
+                     session.session_key,
```

Comment:
Can you leave the overall formatting alone here? (ident 4 spaces, bracket in line with the logger statement.